### ISSUE TITLE
Lacuna Dataset

### ISSUE THREAD

**Issue by sourav-roni** [AdityaGolatkar/SelectiveForgetting#2]
Hi, how should we go forward in replicating the results for Lacuna data-sets mentioned in the paper. Specifically, how to get the data in the place for the experiments?

Thanks in advance

**Comment by YuyangXueEd** [comment]
I think in the paper they mentioned that `Lacuna-10 consists of face images of 10 celebrities from VG- GFaces2, randomly sampled with at least 500 images each. We split the data into a test set of 100 samples for each class, while the remaining form the training set. Similarly, Lacuna-100 randomly samples 100 celebrities with at least 500 images each. We resize the images to 32x32. There is no overlap between the two Lacuna datasets.`

**Comment by d-m-silva** [comment]
Thanks @YuyangXueEd!
